{"uuid":"80cab541-8997-45cd-8588-ba6192970621","historyId":"284145ee999fe0f29929be8a93c9b67d","fullName":"io.cucumber.testng.AbstractTestNGCucumberTests.runScenario","labels":[{"name":"package","value":"testRunner.testRunner"},{"name":"testClass","value":"testRunner.testRunner"},{"name":"testMethod","value":"runScenario"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"testRunner.testRunner"},{"name":"host","value":"MacBookAir.fios-router.home"},{"name":"thread","value":"2332@MacBookAir.fios-router.home.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Runs Cucumber Scenarios","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [User already exist with same contact number] but found [User already exist with same email id]","trace":"java.lang.AssertionError: expected [User already exist with same contact number] but found [User already exist with same email id]\n\tat org.testng.Assert.fail(Assert.java:111)\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\n\tat org.testng.Assert.assertEquals(Assert.java:132)\n\tat org.testng.Assert.assertEquals(Assert.java:656)\n\tat org.testng.Assert.assertEquals(Assert.java:666)\n\tat stepDefinitions.createUserStepDefinition.admin_receives_status_with_contact_message(createUserStepDefinition.java:190)\n\tat âœ½.Admin receives \"409\" \"Conflict\" Status with contact message(file:///Users/mathumathibalakrishnan/eclipse/Mathumathi_2024/2025_Team04_RestAssured_BDD/src/test/resources/features/02_createUser.feature:34)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"arg1","value":"\"Create User API\""},{"name":"arg0","value":"\"Check if Admin is able to create a user with same contact number\""}],"start":1739148175331,"stop":1739148175540}